apis/apps/v1alpha1/componentresourceconstraint_types.go

```diff
@@ -91,6 +97,18 @@ type MemoryConstraint struct {
 	MinPerCPU *resource.Quantity `json:"minPerCPU,omitempty"`
 }
 
+type StorageConstraint struct {
```

Comment:
cannot have all optional members, since this is required attribute in the spec, add validation rules to check need at least specify min or mix

Reply:
fixed, make storage constraint as optional